core/templates/core/user_accounts/account.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'core/base.html' %}
+
+{% block title %}
+  Squire - Account
+{% endblock title %}
+
+{% block content %}
+    Hi {{ user.username }}!
+    This is your account page.
+
+    <br/>
+    This is your email: {{ user.email }}
+
+    <br/><br/>
+    Click <a href="{% url 'core/homepage' %}">here</a> to go back to the homepage.
```

Comment:
Doesn't there need to be a <p> </p> or something similar around this text?

Reply:
All these templates are placeholders; they are ugly (I'd say :P) and are only there to showcase the functionality, which is why not much attention was given to this.

Reply:
Do they work without?

Reply:
Assuming you mean that the template works without the `<p></p>`, yes. HTML does not care much about syntax and accepts pretty much anything. HTML-files can even consist of text only, though that's not really used outside of development/placeholders.